svg/tests/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+add_executable("svg-test"
+    main.cpp)
+
+target_link_libraries("svg-test"
+    PRIVATE "svg" Catch2::Catch2)
+
+# A little hack to include normally private header file 
+# so we can still test some implementation details.
+target_include_directories("svg-test" 
+    PRIVATE ../src
```

Comment:
Should not be needed in your case. If you need it, it means your code will not be usable outside of your library.